services/secondary-storage/server/test/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResourceTest.java

```diff
@@ -49,16 +51,22 @@ public void setUp() {
         resource = new NfsSecondaryStorageResource();
     }
 
+    String filename = "testfile";
+    @After
+    public void cleanup() {
+        File remnance = new File(filename);
+        remnance.deleteOnExit();
```

Comment:
`exists()` check?

Reply:
sure, if that is needed. I didn't expect it to be an issue but we can test that ;)

Reply:
In case someone run in an IDE only 1 of the class tests, it will cause an error. It's best to do the cleanup inside the test method where the file is created.